apps/pkcs12.c

```diff
@@ -221,9 +221,8 @@ int pkcs12_main(int argc, char **argv)
             export_pkcs12 = 1;
             break;
         case OPT_CIPHER:
-            enc_flag = opt_unknown();
-            if (!opt_cipher(enc_flag, &enc))
-                goto opthelp;
+            ciphername = opt_unknown();
+            enc_flag = opt_unknown();   /* overloaded meaning; see below. */
```

Comment:
There doesn't seem to be a below to see for an explanation...

Reply:
I didn't want to add NULL to `opt_md` etc because sometimes there's special behavior if no flag is set.

I pushed a fixup comment to make `enc_flag` more clear.

Reply:
Thanks.